apps/aptos/hooks/useStablePrice.ts

```diff
@@ -22,7 +22,12 @@ export default function useStablePrice(currency?: Currency): Price<Currency, Cur
   const native = useNativeCurrency(chainId)
   const wrapped = currency?.wrapped
   const wnative = native.wrapped
-  const stable = L0_USDC[chainId]
+  const stableArray = [
+    { key: 'lz', coin: L0_USDC[chainId] },
+    { key: 'wh', coin: WH_USDC[chainId] },
+    { key: 'ce', coin: CE_USDC[chainId] },
+  ]
+  const stable = stableArray.find((stableCoin) => currency?.symbol.startsWith(stableCoin.key))?.coin ?? L0_USDC[chainId]
```

Comment:
We should probably need to find all currency-stable pairs and get the most liquidity one.